package/patterns-yast.spec

```diff
@@ -235,21 +235,23 @@ Provides:       pattern-visible()
 
 Recommends:     yast2-devtools
 Recommends:     yast2-testsuite
+Recommends:     osc
+Recommends:     rubygem(%{rb_default_ruby_abi}:yast-rake)
+Recommends:     rubygem(%{rb_default_ruby_abi}:byebug)
+Recommends:     rubygem(%{rb_default_ruby_abi}:rspec)
+Recommends:     rubygem(%{rb_default_ruby_abi}:simplecov)
```

Comment:
do we use simplecov outside of travis?

Reply:
I'm using it locally when writing tests to see the code coverage so it makes sense even outside Travis. Just like Rubocop below...

Reply:
ok